### PR TITLE
Keyboard nav: fix network dropdown

### PR DESCRIPTION
**Pull request by BrodyHughes**
Fixes BX-1066
Before this, the network switcher couldn't be opened with keyboard nav. now it works correctly. The switcher im referring to is the active element in the screenshot below:

Screenshot 2023-10-05 at 4 00 31 PM

### CODE REVIEW

File: src/design-system/components/Lens/Lens.tsx [hunk 1]
```diff
@@ -35,7 +37,7 @@ export function Lens({
         e.key === shortcuts.global.SELECT.key ||
         (e.key === shortcuts.global.FORWARD.key && forwardNav)
       ) {
-        simulateClick(containerRef?.current, { bubbles: false });
+        simulateClick(containerRef?.current, { bubbles: bubbles });
```
Comment (derHowie) [inline src/design-system/components/Lens/Lens.tsx:40]:
```suggestion
        simulateClick(containerRef?.current, { bubbles });
```

File: src/entries/popup/pages/messages/BottomActions/index.tsx [hunk 2]
```diff
@@ -239,6 +239,7 @@ export const BottomNetwork = ({
             gap: 4,
             padding: 2,
           }}
+          bubbles
```
Comment (derHowie) [inline src/entries/popup/pages/messages/BottomActions/index.tsx:242]: bubbles -> bubblesOnKeyDown

**Review by derHowie — CHANGES_REQUESTED** [review]

File: src/design-system/components/Lens/Lens.tsx (hunk 1 above)
Comment (BrodyHughes) [reply src/design-system/components/Lens/Lens.tsx]: with the new descriptive prop name I don't think i can do this

**Review by estebanmino — COMMENTED** [review]
still not working for me https://www.loom.com/share/8ae8bfc33eed4619a194bff4bfdb937f

if i go to click it works
on enter down while the button is tab selected, it doesn't work

**Comment by BrodyHughes** [comment]
> still not working for me https://www.loom.com/share/8ae8bfc33eed4619a194bff4bfdb937f
> 
> if i go to click it works on enter down while the button is tab selected, it doesn't work

Hmm i am seeing it work fine for me, if this is the flow you're referring to (tab down to network, hit enter to open menu)
https://github.com/rainbow-me/browser-extension/assets/41711440/e2d7fc58-8a1c-4a45-b2d2-d3f17ae39373

**Review by estebanmino — APPROVED** [review]
works, not sure what happened yesterday

**Review by greg-schrammel — APPROVED** [review]

**Review by derHowie — APPROVED** [review]

**Pull request merged**